server/data/assessmentClient.ts

```diff
@@ -14,7 +14,14 @@ export default class AssessmentClient {
     return (await this.restClient.get({ path: paths.assessments.index.pattern })) as Array<Assessment>
   }
 
-  async find(applicationId: string): Promise<Assessment> {
-    return (await this.restClient.get({ path: paths.assessments.show({ id: applicationId }) })) as Assessment
+  async find(assessmentId: string): Promise<Assessment> {
+    return (await this.restClient.get({ path: paths.assessments.show({ id: assessmentId }) })) as Assessment
+  }
+
+  async update(assessment: ApprovedPremisesAssessment): Promise<ApprovedPremisesAssessment> {
+    return (await this.restClient.put({
+      path: paths.assessments.update({ id: assessment.id }),
+      data: assessment.data,
```

Comment:
This should be:

```suggestion
      data: { data: assessment.data },
```

As per the [API spec](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/src/main/resources/static/api.yml#L3194)